rero_ils/modules/selfcheck/api.py

```diff
@@ -375,24 +375,26 @@ def selfcheck_checkout(transaction_user_pid, library_pid, patron_barcode,
         return checkout
 
 
-def selfcheck_checkin(transaction_user_pid, library_pid, patron_barcode,
+def selfcheck_checkin(transaction_user_pid, institution_id, patron_barcode,
```

Comment:
` institution_id` and  `patron_barcode` are not used in this function ?

Reply:
ok, it's changed